test/Google.Api.Gax.Grpc.Tests/CallSettingsTest.cs

```diff
@@ -95,6 +95,28 @@ public void ToCallOptions_All()
         }
 
         [Fact]
+        public void Empty()
+        {
+            var empty = CallSettings.Empty;
+            Assert.Null(empty.CancellationToken);
+            Assert.Null(empty.Credentials);
+            Assert.Null(empty.PropagationToken);
+            Assert.Null(empty.HeaderMutation);
+            Assert.Null(empty.Timing);
+            Assert.Null(empty.WriteOptions);
+        }
+
+        [Fact]
+        public void Empty_Merge()
+        {
+            var overlay = CallSettings.FromCancellationToken(CancellationToken.None);
```

Comment:
I'd hoped that this *would* return `CallSettings.Empty`, but of course specifying a cancellation token of none isn't the same as omitting it, as it can override one. Urgh.

Presumably we can have a second singleton which is for CancellationToken.None?